cookbook/getting-started/appbar-basics.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: page
+title: "AppBar Basics"
+---
+
+Todo: Rewrite this with more detail / less complex state management 
+
+-----
+
+A typcial AppBar with a title, actions, and an overflow dropdown menu.
+
+<p>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-md-4">
+        <div class="panel panel-default">
+          <div class="panel-body" style="padding: 16px 32px;">
+            <img style="border:1px solid #000000" src="https://storage.googleapis.com/flutter-catalog/cb4a54db8fb3726bf4293b9cc5cb12ce16883803/basic_app_bar_small.png" alt="Android screenshot" class="img-responsive">
+          </div>
+          <div class="panel-footer">
+            Android screenshot
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</p>
+
+An app that displays one of a half dozen choices with an icon and a title.
+The two most common choices are available as action buttons and the remaining
+choices are included in the overflow dropdow menu.
+
+Try this app out by creating a new project with `flutter create` and replacing the contents of `lib/main.dart` with the code that follows.
```

Comment:
wrap at 80 cols

Reply:
Cool, should I document that for other contributors? Most .md files don't wrap at 80.